test/test-browser.js

```diff
@@ -26,6 +26,6 @@ if (window.browserTest) {
 }
 
 require('probe.gl'); // ensure probe gets access to pure console first
-require('tap-browser-color')();
+// require('tap-browser-color')();
```

Comment:
What happens here? Do we no longer print console in the browser window? I understand that we stream it to the console, but we could still show it in the browser?

Reply:
`tap-browser-color` directly accesses `document.body` which becomes `null` during `puppeteer.screenshot`. I didn't find a way to make the image diffing tests NOT run in parallel with others.